pre_commit/git.py

```diff
@@ -36,7 +38,12 @@ def no_git_env(_env=None):
 
 
 def get_root():
-    return cmd_output('git', 'rev-parse', '--show-toplevel')[1].strip()
+    return fix_mingw_path(
+        cmd_output(
+            'git', 'rev-parse',
+            '--show-toplevel',
+        )[1].strip(),
+    )
```

Comment:
I don't think this should happen, if there's adjustments happening here it sounds like you're in the mismatch case (where you've got cygwin something and windows something else) -- same goes for the `hook-tmpl`, though sadly we can't warn early enough there :/